packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/TableRows/index.js

```diff
@@ -25,7 +25,7 @@ const TableRows = ({ headers, rows, onOpenModal }) => {
           id: `Settings.permissions.auditLogs.${value}`,
           defaultMessage: getDefaultMessage(value),
         },
-        { model }
+        { model: `(${model})` }
```

Comment:
ℹ️  I tried https://formatjs.io/docs/core-concepts/icu-syntax/#select-format but couldn't get it to work... 🤷‍♂️

Reply:
This is okay but yes the select thing would be cleaner. Maybe it's because of a version issue? The docs are for the latest react-intl v6, we have v5

Reply:
Imo the select syntax fully works and we have used that recently in https://github.com/strapi/strapi/pull/15424

Also I've bumped react-intl to v6 in https://github.com/strapi/strapi/pull/15625

Reply:
ok thanks, I'll take a look and see why I wasn't able to get it working. In the end I opted for a function to concatenate the model instead. You think the select would be better?
 https://github.com/strapi/strapi/blob/69df8f6bfedced0d2116aba79ed083441bd2895d/packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/utils/formatMessage.js#L36

Reply:
Yeah I think so :)